catalog/content/core/content-core-api/src/main/java/ddf/content/util/Describable.java

```diff
@@ -21,7 +21,8 @@
  * @author Hugh Rodgers, Lockheed Martin
  * @author ddf.isgs@lmco.com
  */
-public interface Describable {
+@Deprecated
+public interface Describable extends org.codice.ddf.platform.api.Describable {
```

Comment:
If we are deprecating this we should update all of it's implementors to use the new interface.